examples/selectors/README.md

```diff
@@ -42,3 +43,49 @@ Creates an EventListener that serve triggers in multiple namespaces.
    ```bash
    tkn pr -n bar list
    ```
+
+## Label Selector EventListener
+
+Creates an EventListener that serve triggers selected via a label selector.
+
+### Try it out locally:
+
+1. To create the label selector trigger and all related resources, run:
+
+   ```bash
+   kubectl apply -f examples/selectors/
```

Comment:
if we apply like this everything get creates.
label selector is not working because namespace `baz` is not provided in rbac.yaml 

How about if we do something like below for clear understanding 

```
[selectors]$ tree
.
├── label
│   ├── 00_ns.yaml
│   ├── 01_rbac.yaml
│   ├── 04_eventlistener-label-sel.yaml
│   └── 05_labeled-triggers.yaml
├── namespace
│   ├── 00_ns.yaml
│   ├── 01_rbac.yaml
│   ├── 02_eventlistener-sel.yaml
│   └── 03_trigger.yaml
└── README.md
```
and adding instruction in one README.md file

Reply:
I'll update to this structure, thanks

Reply:
@savitaashture in the `label` directory, I named the files 00, 01, 02_eventlistener-label-sel, 03 which matched the structure in the namespace directory. Then, in the `namespace` directory, I called the 02 file `02_eventlistener-ns-sel`